src/metabase/pulse/render/body.clj

```diff
@@ -616,43 +607,15 @@
                                 :width   :100%})
            :src   (:image-src image-bundle)}]]})
 
-(def ^:private axis-group-threshold 0.33)
-
-(defn- render-multiple-lab-chart
-  "When multiple non-scalar cards are combined, render them as a line, area, or bar chart"
-  [render-type card dashcard {:keys [viz-settings] :as data}]
-  (let [viz-settings  (merge viz-settings (:visualization_settings dashcard))
-        multi-res     (pu/execute-multi-card card dashcard)
-        ;; multi-res gets the other results from the set of multis.
-        ;; we shove cards and data here all together below for uniformity's sake
-        viz-settings  (set-default-stacked viz-settings card)
-        cards         (cons card (map :card multi-res))
-        multi-data    (cons data (map #(get-in % [:result :data]) multi-res))
-        rowfns        (mapv common/graphing-column-row-fns cards multi-data)
-        row-seqs      (for [[row-seq [x-rowfn y-rowfn]] (map vector (map :rows multi-data) rowfns)]
-                        (map (juxt x-rowfn y-rowfn)
-                             (common/row-preprocess x-rowfn y-rowfn row-seq)))
-        col-seqs      (map :cols multi-data)
-        first-rowfns  (first rowfns)
-        [x-col y-col] ((juxt (first first-rowfns) (second first-rowfns)) (first col-seqs))
-        labels        (x-and-y-axis-label-info x-col y-col viz-settings)
-        names         (map :name cards)
-        colors        (take (count multi-data) colors)
-        types         (replace {:scalar :bar} (map :display cards))
-        settings      (->ts-viz x-col y-col labels viz-settings)
-        y-pos         (take (count names) (default-y-pos data axis-group-threshold))
-        series        (join-series names colors types row-seqs y-pos)]
-    (attach-image-bundle (image-bundle/make-image-bundle render-type (js-svg/combo-chart series settings)))))
-
 (defn- multiple-scalar-series
   [joined-rows _x-cols _y-cols _viz-settings]
-  ;; TODO: Extra vars could be used for color settings
-  (for [[idx row-val] (map-indexed vector joined-rows)]
-    {:name  (first row-val)
-     :color (nth colors idx)
-     :type  :bar
-     :data [row-val]
-     :yAxisPosition "left"}))
+  [(for [[row-val] (map vector joined-rows)]
```

Comment:
I think the chart is still weird when only has 2 bars since they're not centered on the tick. As seen in [this comment](https://github.com/metabase/metabase/pull/25811#issuecomment-1277599092)
> Got it working with scalar / bar charts now, both with one extra card and with multiple:
> 
> ![image](https://user-images.githubusercontent.com/784417/195607561-4140107b-b0ff-426a-92c5-644bf341894e.png)

This is from my testing with different versions
#### App
![image](https://user-images.githubusercontent.com/1937582/204447017-1061940f-b506-4eca-8f4e-0b69bc2eb92e.png)

#### v.0.44.6
![image](https://user-images.githubusercontent.com/1937582/204446943-2484f836-5057-4d10-b892-ee834f5a1844.png)


#### Master
![image](https://user-images.githubusercontent.com/1937582/204446962-0cd717ec-a0c2-4db9-bf95-3b7c9034ca79.png)


#### This branch
![image](https://user-images.githubusercontent.com/1937582/204446978-6bfe7beb-070d-4e28-9a75-bc30c07d844a.png)